src/test/java/NotifierTest.java

```diff
@@ -0,0 +1,55 @@
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.CoreMatchers.containsString;
+
+/**
+NotifierTest ensures that we notify the user of generated reports.
+@author Erik Cole
+@author Chris Waldon
+@version 0.1
+*/
+public class NotifierTest {
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final PrintStream realStdOut = System.out;
+
+    /**
+    This redirects stdout into a stream that we can evaluate.
+    */
+    @Before
+    public void stealStdOut() {
+        System.setOut(new PrintStream(outContent));
+    }
+
+    /**
+    This restores stdout to the default.
+    */
+    @After
+    public void resetStdOut() {
+        System.setOut(realStdOut);
+    }
+
+    /**
+    This checks that the e2e test notifies the user of where
+    the report was created.
+    */
+    @Test
+    public void checkForMessage() {
+        ScattTest.testEndToEnd("GoodSb2Dir", 0b111_1111);
```

Comment:
It might be most readable to call 0b111_1111 something like ALL_FEATURES_2017_04_19.  That would make sense of why we are using that particular number and everything.